MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -56,15 +56,15 @@
                                no_track_cuts = cms.bool(False),
                                min_seed_jet_pt = cms.double(30),
                                min_track_pt = cms.double(1),
+                               min_leptrack_pt = cms.double(20), # should already be enforced during earlier steps; just as precaution/placeholder
                                min_track_pt_loose = cms.double(0.9), # loose track selection when match_jets is True
                                min_track_dxy = cms.double(0),
                                min_track_sigmadxy = cms.double(0),
                                min_track_rescaled_sigmadxy = cms.double(4),
-                               min_leptrack_rescaled_sigmadxy = cms.double(3),
+                               min_leptrack_rescaled_sigmadxy = cms.double(3), # relaxed lepton nsigmadxy 
                                min_track_rescaled_sigmadxy_loose = cms.double(3.5), # loose track selection when match_jets is True
                                min_track_sigmadxypv = cms.double(0),
-                               min_track_hit_r = cms.int32(1),
-                               min_leptrack_hit_r = cms.int32(2),
+                               min_track_hit_r = cms.int32(1), # relaxed to 2 iff lost hits = 0 
```

Comment:
Does the comment mean that the C++ handles that part? Or is it a note of something to modify? Just want to be sure!!

Reply:
Tagging @abbywarden  here